clusterloader2/pkg/prometheus/experimental.go

```diff
@@ -130,8 +130,9 @@ func (pc *PrometheusController) snapshotPrometheusDiskIfEnabled() error {
 
 func (pc *PrometheusController) trySnapshotPrometheusDisk(pdName, snapshotName, zone string) error {
 	klog.Info("Trying to snapshot Prometheus' persistent disk...")
-	klog.Infof("Snapshotting PD %q into snapshot %q in zone %q", pdName, snapshotName, zone)
-	cmd := exec.Command("gcloud", "compute", "disks", "snapshot", pdName, "--zone", zone, "--snapshot-names", snapshotName)
+	project := pc.clusterLoaderConfig.PrometheusConfig.SnapshotProject
+	klog.Infof("Snapshotting PD %q into snapshot %q in project %q in zone %q", pdName, snapshotName, project, zone)
```

Comment:
Please add a check that project != "". Sth like.
```
if project == "" {
  // This should never happen when run from prow - kubetest should always propagate PROJECT env var.
  return fmt.Errorf("unknown project - please set --experimental-snapshot-project flag")
}
```

Reply:
Done.